dashboard/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import Mainbody from './Mainbody';
+import Sidenav from './Sidenav';
+
+
+
+ReactDOM.render(
+  [
+    <Sidenav/>,
```

Comment:
Make it render one root component instead of two

Reply:
I will change it and i will update it